camviewer.ui

```diff
@@ -898,6 +898,30 @@ Vmin\</string>
            </property>
           </widget>
          </item>
+         <item row="2" column="3">
+          <widget class="QLineEdit" name="lineEditRangeMax">
```

Comment:
Any reason we shouldn't make this a QSpinBox?

Reply:
No reason, a QSpinBox would be strictly better. It would let us constrain the min/max values more precisely, allow us to stop doing bonus str <-> int conversions ourselves, and has built-in input validation. I can swap it over.

Reply:
I think this is also a rare case where I might like the spinbox arrows. Putting the ranges up or down in a few clicks could be nice.